### ISSUE TITLE
Godot Linux Releases Inconsistently Named

### ISSUE THREAD

**Issue by b4ux1t3** [godotengine/godot-builds#5]
### Godot version

All

### System information

Irrelevant

### Issue description

The releases for Godot and Godot's .NET (Mono) Linux versions are inconsistently named.

The non-Mono version is named `Godot_v4.1.3-stable_linux.x86_64.zip`, with a `.` (dot) between `linux` and the architecture, `x86_64`. 

The Mono version uses an underscore in place of that dot: `Godot_v4.1.3-stable_mono_linux_x86_64.zip`.

Additionally, the mono version is nested within a directory within the zip file, whereas the non-Mono version is at the root of the zip.

Look, I know this is a silly little thing. I'm developing a CI pipeline template for Godot projects, and it'd remove about ten lines of bash if you guys fixed this.

### Steps to reproduce

See the attached screenshot, or look at the [Releases page](https://github.com/godotengine/godot/releases): 
![image](https://github.com/godotengine/godot/assets/5150833/e76b3e15-7c25-4273-b8f0-9c4da35851b0)

### Minimal reproduction project

N/A

**Comment by akien-mga** [comment]
Moved to godot-builds as it's not an engine bug.

**Comment by b4ux1t3** [comment]
Ah. I see, that makes perfect sense. Thanks for moving it, too, I didn't realize it was a different repo.

I guess the only fix would be to either put the non-Mono version in a nested folder (kind of silly), or to bring the contents of the nested folder in the Mono version up one level (potentially not doable?).

I can see how that might break existing workflows around this pretty thoroughly.

The consistency's not THAT important to me. I'll leave this open and let you guys figure out if the juice is worth the squeeze. If not, I'm totally cool if you close it.

**Comment by akien-mga** [comment]
Yeah it's an inconsistency that also bothered me since the beginning, but I'm not sure yet what's the right solution.

We plan to rework the whole distribution in the future, to make it easier to download export templates for a single platform on demand, so this might be part of that new setup and we'll try to have something consistent for Linux editor builds too.

**Comment by b4ux1t3** [comment]
Ah, yeah. I've built a bash function that will download the correct export templates, but I keep thinking it'd be cool if the godot executable could do that for me (like the editor does). Even if that's not your goal, anything that lets me remove lines of bash is a win in my book.
